Mixpanel/MPObjectSelector.h

```diff
@@ -6,7 +6,7 @@
 //  Copyright (c) 2014 Mixpanel. All rights reserved.
 //
 
-#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
```

Comment:
This header file doesn't use anything from UIKit; Foundation should be sufficient. (You might need to add UIKit somewhere else, though.)